### PR TITLE
Migrate Paparazzi to layoutlib 4.2

### PR DESCRIPTION
**Pull request by jrodbx**

### CODE REVIEW

**Review by geoff-powell — APPROVED** [review]
🎉

File: paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt [hunk 1]
```diff
@@ -260,7 +263,25 @@ class Paparazzi(
...
+      val currentTimeMs = SystemClock_Delegate.uptimeMillis()
+      val choreographerCallbacks =
+        RenderAction.getCurrentContext().sessionInteractiveData.choreographerCallbacks
+      choreographerCallbacks.execute(currentTimeMs, Bridge.getLog())
+    } catch (e: Throwable) {
+      Bridge.getLog().error("broken", "Failed executing Choreographer#doFrame", e, null, null)
+      throw e
+    }
+
     try {
```
Comment (JakeWharton) [inline paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt:285]: This should start at line 266

**Review by JakeWharton — APPROVED** [review]

File: paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt [hunk 2]
```diff
@@ -129,7 +132,7 @@ class Paparazzi(
         .copy(
             layoutPullParser = LayoutPullParser.createFromString(contentRoot),
             deviceConfig = deviceConfig,
-            renderingMode = SessionParams.RenderingMode.V_SCROLL
+            renderingMode = SessionParams.RenderingMode.NORMAL
```
Comment (jrodbx) [inline paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt:135]: this didn't work until layoutlib 4.2, and now results infinitely scrolling lists, which may not be desirable by default.  cc: @gamepro65 who has a pending fix to expose this configuration to test authors.

**Comment by jrodbx** [comment]
i'm really confused why this isn't passing on CI now.  gonna run a git bisect.

**Comment by jrodbx** [comment]
Odd.  All of a sudden, ubuntu builds are failing, yet macos builds seem to work, with respect to animations

**Comment by jrodbx** [comment]
Not the best, but looks like pinning to a non-latest version of Ubuntu works, per this hint from SqlDelight: https://github.com/cashapp/sqldelight/blame/master/.github/workflows/PR.yml#L13-L18

**Pull request merged**